File: Day-45/Purvi.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        l1, l2=[], []
        for i in range(n):
            l1.append(nums[i])
            l2.append(nums[n+i])
        ans=[]
        for i in range(n):
            ans.append(l1[i])
            ans.append(l2[i])
        return ans

File: Day-57/Purvi.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        int i, j;
        vector<int>a(nums.size(), 0);
        for(i=0; i<nums.size(); i++)
            for(j=0; j<nums.size(); j++)
            {
                if(nums[j]<nums[i])
                    a[i]++;
            }
        
        return a;
    }
};